tools/loadtest/README.md

```diff
@@ -0,0 +1,48 @@
+# Fulcio Performance Test
+
+## Overview
+
+[Learn more about Locust](http://docs.locust.io/en/stable/index.html).
+
+1. Install Locust with `pip3 install -r requirements.txt`
+1. Fetch an identity token for a service account with `gcloud auth print-identity-token --audiences sigstore --impersonate-service-account <name>@<project-id>.iam.gserviceaccount.com --include-email`.
```

Comment:
looks cool, just a small comment

I would add a requirements section to let the users know what is needed to have and set up beforehand.

like,
- gcp project
- software dependencies (which python version...)

Reply:
Good idea, added a prereqs section!